llvm/test/MC/AMDGPU/hsa-diag-v4.s

```diff
@@ -86,7 +86,6 @@
 .end_amdhsa_kernel
 
 // ALL-LABEL: warning: test_amdhsa_accum_offset_missing
-// NONGFX9A: error: directive requires gfx90a+
```

Comment:
I had to remove this one to get the tests to pass. I did not look into this too deeply.

Reply:
Usually these are we missed a check for error somewhere earlier in the file. Really this test should be using --implicit-check-not=error to prevent this from happening